src/app/store.jsx

```diff
@@ -0,0 +1,19 @@
+import { createStore , compose } from 'redux';
+import { syncHistoryWithStore } from 'react-router-redux';
+import { browserHistory } from 'react-router';
+
+import rootReducer from './reducers/index';
+
+//Create default data
+
+
+const defaultState = {
+	organizations: {selected: null , items: []} ,
+	assemblies: {organization: null , items: []} //current assembly
+}
+
+const store = createStore(rootReducer , defaultState);
+
+export const history = syncHistoryWithStore(browserHistory, store);
```

Comment:
How are we using browser history?  Are we thinking of refactoring the app to use urls to navigate?